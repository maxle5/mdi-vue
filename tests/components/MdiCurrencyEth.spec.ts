
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiCurrencyEth from "../../src/components/MdiCurrencyEth.vue";

test("MdiCurrencyEth snapshot", () => {
  const wrapper = mount(MdiCurrencyEth, {});
  expect(wrapper.html()).toMatchSnapshot();
});
