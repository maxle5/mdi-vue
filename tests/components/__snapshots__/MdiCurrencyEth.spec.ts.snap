// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiCurrencyEth snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M6,5H18V7H6M7,11H17V13H7M5.5,17H18.5V19H5.5" style="fill: currentColor;"></path>
</svg>"
`;
